### PR TITLE
README update for alpha release

### LINKED ISSUE #356: Commande line example from README does not work
Hi,

I've started to use prettier-php so I've followed the README example. When I run from my ubuntu 16 cli `prettier --write ...` I get an error `Command not found` as the library is installed locally.

I've tried with `./node_modules/.bin/prettier --write ...` and then it works. Would it be better to use as a global cli tool, so `prettier` would be available ?

Also, after running the command it was also running over the `vendor` folder, maybe a `./node_modules/.bin/prettier --write "src/**/*"` would be safer

### PR DESCRIPTION
**Pull request by czosel**
Fixes #356

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,37 +34,39 @@
...
-* Create a file called `test.php`.
-* Run `yarn prettier test.php` to check the output.
+Please note that this plugin is currently in alpha stage and still under active development. We encourage everyone to try it and give feedback, but we don't recommend it for production use yet.
 
 ## Install
 
 yarn:
+
 ```bash
 yarn add --dev prettier/prettier prettier/plugin-php
```
Comment (czosel) [inline README.md]: Or should we prefer global installation?

Comment (evilebottnawi) [reply README.md]: @czosel Let's do this as do `prettier` here https://prettier.io/docs/en/install.html

File: README.md [hunk 2]
```diff
@@ -34,37 +34,61 @@
...
+```json
+scripts: {
+  "prettier":"prettier"
+}
+```
+
+and then run it via
+
+```bash
+yarn run prettier path/to/file.php
```
Comment (mgrip) [inline README.md]: still need the `--write` here

Comment (czosel) [reply README.md]: :+1: fixed

File: README.md [hunk 3]
```diff
@@ -34,37 +34,61 @@
...
+npm install --global prettier @prettier/plugin-php
 ```
 
 ## Use
 
+If you installed prettier as a local dependency, you can add prettier as a script in your `package.json`,
+
+```json
+scripts: {
+  "prettier":"prettier"
```
Comment (evilebottnawi) [inline README.md]: Do we need glob (`{**/*,*}.php`) here?

Comment (czosel) [reply README.md]: Not when you're calling it like instructed below: `yarn run prettier path/to/file.php --write`

**Comment by czosel** [comment]
I just tried the local installation with `prettier/prettier` and `prettier/plugin-php`. After the release, it should work with `@prettier/plugin-php`. I'd say let's replace `prettier` with `prettier/prettier` for now, and do the release :wink:

**Comment by czosel** [comment]
Alright, let's do this :smile: 

Most important question: What are we calling the version? `0.1`? `0.1-alpha`?

**Comment by mgrip** [comment]
My vote would be `0.1.0` I think? https://semver.org/#how-should-i-deal-with-revisions-in-the-0yz-initial-development-phase

**Comment by mgrip** [comment]
@czosel I'd say feel free to publish on npm and update the package.json as part of this PR

**Comment by czosel** [comment]
Done :tada: 
https://www.npmjs.com/package/@prettier/plugin-php

**Comment by mgrip** [comment]
Amazing 🎉 awesome work guys!

**Comment by azz** [comment]
Awesome! 🎉

**Pull request merged**